src/node/entry.ts

```diff
@@ -90,6 +90,7 @@ const main = async (args: Args): Promise<void> => {
   } else {
     logger.info("  - No authentication")
   }
+  process.env.PASSWORD = undefined
```

Comment:
This doesn't really matter but since this will set it to the literal string "undefined" should we do `delete process.env.PASSWORD` instead?

Reply:
Forgot, it's so dumb...